### DISCUSSION TITLE
[Question] MenuItem height limit

### DISCUSSION THREAD

**Discussion by WackyIdeas** [tsujan/Kvantum#806] [Q&A] [answered]
Hello. I've been configuring a theme for Kvantum and I've been really enjoying the capabilities that Kvantum has to offer compared to other Qt styles. However, I came across an obstacle when it comes to MenuItem's height. It appears that there's no way to reduce the height of this element as it caps at 25 pixels. Is there any way to reduce this height further, or is this a coded/designed limit for Kvantum? Other themes that have settings that make widgets more compact also have the hard 25 pixel limit. 

The configuration options I have set to 0 for MenuItem are: text.margin.top, text.margin.bottom, frame.top, frame.bottom. Also, frame.top and frame.bottom being set to 0 does not reduce the height further, as it simply gets rid of the frame border altogether at the top and bottom. 

Is there a way to reduce the height past 25 pixels, or is this a deliberate choice set in Kvantum? Thank you in advance. 

![image](https://user-images.githubusercontent.com/60061632/175525514-1d8c4a22-74fc-4b2a-a6b4-4591ff4c309a.png)

**Comment by tsujan** [comment]
See `Theme-Config → menu_separator_height` and `Theme-Making.pdf → Indicators → MenuItem`

**Comment by WackyIdeas** [comment, marked as answer]
While that didn't solve the problem for me, I ended up doing a svg 'hack' with gradients to reduce the height to 21px, which produced the desirable results. 
![image](https://user-images.githubusercontent.com/60061632/175780287-4ca1fec4-15db-407e-b9b5-4bb500003ee4.png)

**Reply by tsujan** [reply]
That isn't menu-item height.
